python_coreml_stable_diffusion/activation_quantization.py

```diff
@@ -0,0 +1,491 @@
+#
```

Comment:
I think it would make sense to add a small readme on how to use this script, with example commands.  
I guess it has to be used in a couple of steps, first to generate calibration data and then use that data to get the quantized model. 

It could be a separate readme accompanying this file or a new short section in the main readme (like [this](https://github.com/apple/ml-stable-diffusion/tree/main?tab=readme-ov-file#-advanced-weight-compression-lower-than-6-bits) one)